en/docs/security-announcements/cve-justifications/2024/CVE-2022-29464.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "CVE-2022-29464"
+category: security-announcements
+date: August 21, 2024
+---
+
+# CVE-2022-29464
+
+<p class="doc-info">WSO2 Products impacted: no</p>
+<p class="doc-info">Customers actions required: no</p>
+---
+
+### REPORTED VULNERABILITY
+
+The CVE-2022-29464 vulnerability was reported as affecting certain WSO2 components. However, the initial CVE publication[^1] does not contain accurate **Product Status** information, and external advisories, such as the one on NVD[^2], incorrectly identify vulnerable components.
+
+### WSO2 JUSTIFICATION
+
+The vulnerability mentioned in CVE-2022-29464 has been addressed and fixed in the respective versions as per the advisory published at WSO2-2021-1738[^3]. The affected components and their corresponding fixed versions are listed below:
+
+### Affected Components and Fixed Versions:
+
+#### 1. WSO2 Carbon UI (`org.wso2.carbon:org.wso2.carbon.ui`)
+
+Unaffected ranges are,
+
+| version (start range) | <= version (range) | Version Type |
+|-----------------------|--------------------|--------------|
+| 4.4.9.8               | 4.4.9.*            | Commercial   |
+| 4.4.20.6              | 4.4.20.*           | Commercial   |
+| 4.4.26.10             | 4.4.26.*           | Commercial   |
+| 4.4.32.9              | 4.4.32.*           | Commercial   |
+| 4.4.36.6              | 4.4.36.*           | Commercial   |
+| 4.4.40.14             | 4.4.40.*           | Commercial   |
+| 4.5.1.16              | 4.5.1.*            | Commercial   |
+| 4.5.3.17              | 4.5.3.*            | Commercial   |
+| 4.6.0.77              | 4.6.0.*            | Commercial   |
+| 4.6.1.35              | 4.6.1.*            | Commercial   |
+| 4.6.2.42              | 4.6.2.*            | Commercial   |
+| 4.7.0                 | *                  | Community[^4][^5]    |
+
```

Comment:
Seems version rage 4.4.22.*, and 4.4.35.* are missing

Reply:
Thanks for highlighting. Added the information. Could you please check it now.